README.md

```diff
@@ -4,89 +4,42 @@ Water The Trees is a platform that crowd sources tree planting and maintenance.
 
 ## Join Us!
 
-https://docs.google.com/document/d/1L5Hc8_K_NhVhAejdE05C_Y__CgqeBWFrFYFoNqBSBbQ/edit
-Water the Trees is an open source project run by tree planting volunteers at Sierra Club, Public Works, and developers at Code for America.
+Water the Trees is an open source project run by tree planting volunteers and developers at Code for America.
 
 ### INSTALL GIT
 
 1. install git https://git-scm.com/downloads
-
-### INSTALL DOCKER
-
-1. install docker https://www.docker.com/get-started
-2. install docker-compose (only on linux, on osx this installs with docker)
-
-### CLONE REPO
-
-1. Open terminal and run this
-
+2. Install node with nvm
 ```shell
-$ git clone https://github.com/waterthetrees/waterthetrees.git
-$ cd waterthetrees
+$ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.2/install.sh | bash
```

Comment:
Nitpick but could you remove the `$` in the shell commands. I understand it's to show that this is a terminal command, but it would be nice to be able to just click the copy button, without having to manually copy the line except the `$`.